tests/__snapshots__/test_jinja.ambr

```diff
@@ -1,15 +1,142 @@
 # serializer version: 1
 # name: test_render_recipe_with_context
   '''
+  about:
+    description: '# Mamba, the Fast Cross-Platform Package Manager
+  
+      ${{ env.get(&#34;MY_ENV_VAR&#34;)}}
```

Comment:
Is the escaping correct here?

Reply:
it is quotation mark, I'm not exactly sure why `syrupy` keep it like this in snapshots

Reply:
fixed by asking jinja to not escape it :)